Dockerfile

```diff
@@ -1,8 +1,4 @@
-FROM golang:1.8.3-alpine
-
-MAINTAINER Ed Rooth <ed.rooth@coreos.com>
-MAINTAINER Lucas Servén <lucas.serven@coreos.com>
-MAINTAINER Rithu John <rithu.john@coreos.com>
+FROM golang:1.9.2-alpine
```

Comment:
why remove the maintainers list?

Reply:
Sorry should have called that out.

I'd like to clean up the top level MAINTAINER list soon. I don't really see the point of having this list here as well. Our container's don't need our emails in them.

Either way, I can revert this change if we'd like to keep these.

Reply:
Its fine to take them out here as long as we maintain the top level one and update the list there.

Reply:
I'll revert the change from this PR for now